server/models/Category.js

```diff
@@ -0,0 +1,19 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+// create ninja Schema & model
+const addCategorySchema = new Schema({
+    title: {
+        type: String,
+    },
+    description: {
+        type: String
+    },
+    shortDescription: {
+        type: String
+    }
+});
+
+const AddCategory = mongoose.model('addCategories', addCategorySchema);
```

Comment:
the name of the model is what ends up being the database collection - name it "Category"